### PR TITLE
issue-5230 add traceid and spanid to decision log.

### PR DESCRIPTION
**Pull request by be-a-bee**

### CODE REVIEW

File: internal/internal_test.go [hunk 1]
```diff
@@ -49,6 +49,8 @@ const exampleAllowedRequest = `{
 			"x-b3-sampled": "1",
 			"x-b3-spanid": "537f473f27475073",
 			"x-b3-traceid": "537f473f27475073",
+			"traceparent" : "00-0000000000000000f7bea0490c3cb268-b9543a567b0289f8-01",
+			"tracestate" : "rojo=00f067aa0ba902b7",
```
Comment (ashutosh-narkar) [inline internal/internal_test.go:53]: If you're not using this, please remove.

**Review by ashutosh-narkar — COMMENTED** [review]
Hey @be-a-bee just one small comment. We can then merge this. Thanks!

**Comment by be-a-bee** [comment]
@ashutosh-narkar  , thank you. addressed your comment. please review

File: test/e2e/distributedtracing/distributedtracing_test.go [hunk 2]
```diff
@@ -147,5 +148,52 @@ func TestServerSpan(t *testing.T) {
 		if got, expected := spans[0].Parent.SpanID(), parentSpanID; got != expected {
 			t.Errorf("expected span to be child of %v, got parent %v", expected, got)
 		}
+
+		var entry test.LogEntry
+		var found bool
+
+		for _, entry = range consoleLogger.Entries() {
+			//t.Log(entry.Message)
```
Comment (ashutosh-narkar) [inline test/e2e/distributedtracing/distributedtracing_test.go:156]: Nit: We should remove this.

**Review by ashutosh-narkar — COMMENTED** [review]
The changes lgtm. Just one small comment. Thanks @be-a-bee!

**Comment by be-a-bee** [comment]
@ashutosh-narkar , thank you for your prompt response. 

I addressed it. please take a look

**Review by ashutosh-narkar — APPROVED** [review]
Thanks for the contribution @be-a-bee!

**Pull request merged**